src/components/fhir/FhirContext.tsx

```diff
@@ -26,9 +26,17 @@ export const FhirProvider: FC<Props> = (props) => {
   useEffect(() => {
     if (!client) {
       SMART.ready()
-        .then((client) => setClient(client))
-        .catch((error) => setError(error))
-        .finally(() => console.debug("FHIR client ready"));
+        .then((client) => {
+          setClient(client);
+        })
+        .catch((error) => {
+          setError(error);
+          console.error(error);
+        })
+        .finally(() => {
+          console.debug("FHIR client ready");
+          setError(null);
```

Comment:
do we want this error null here finally? I would have thought that this will reset the error state after the .catch executes?

Reply:
Yes I'm not quite done with this yet. I'm exploring a few approaches with providing context to the application. I've converted the PR to draft 👍